docs/content/common/azkfilejs/php-cakephp.md

```diff
@@ -0,0 +1,31 @@
+```js
+systems({
+  'my-laravel-app': {
```

Comment:
thanks for the contribution, @lucas-macedo.

I will test your PR so that we can already do the merge. Meanwhile, you could make a little adjustment?

where says `my-laravel-app` switch to` my-cake-app`?